src/frontend/scss/colors/_theme.scss

```
@@ -277,6 +277,7 @@ $r-theme: (
     sub-title-color: r-color('slate-grey'),
     active-color: r-color('indianred3'),
     base-border: r-color('light-grey'),
+    menu-link-left-spacing: 25px,
```

I'll use `padding` instead of `spacing` to give provide more information.

```suggestion
    menu-link-left-padding: 25px,
```